frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/cater-capacity/cater-capacity.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngIf="canCaterFor" style="max-width: 20vw">
+    <span *ngIf="type === 'material'">Dieses Material </span>
+    <span *ngIf="type === 'personnel'">Diese Einsatzkraft </span>
```

Comment:
You can use a ternary operator for this: 

```suggestion
    <span>{{ type === 'material'? 'Dieses Material' : 'Diese Einsatzkraft' }}</span>
```

Reply:
Sure